build.gradle

```diff
@@ -1,47 +1,21 @@
-//////////////////////////////
-// Buildscript Dependencies //
-//////////////////////////////
-buildscript {
-    repositories {
-        jcenter()
-        maven { url 'https://palantir.bintray.com/releases/' }
-    }
-
-    dependencies {
-        classpath 'com.netflix.nebula:gradle-info-plugin:7.1.4'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.5'
-        classpath 'com.netflix.nebula:nebula-publishing-plugin:17.2.1'
-    }
-}
-
 /////////////
 // Plugins //
 /////////////
 plugins {
     id 'com.palantir.git-version' version '0.12.3'
     id 'org.inferred.processors' version '3.3.0'
-}
-
-apply plugin: 'java'
-apply plugin: 'idea'
+    id 'com.palantir.external-publish' version '0.4.0'
+    id 'com.palantir.external-publish-jar' version '0.4.0'
```

Comment:
latest version is 1.2.1

Reply:
Oh wow, ok. I was just going off the excavator version, but I guess that was quite a while ago. Will update.